examples/react/react-native-app/ios/Podfile

```diff
@@ -2,7 +2,7 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 require_relative '../node_modules/@nrwl/react-native/nx_post_install'
 
-platform :ios, '11.0'
+platform :ios, '13.0'
```

Comment:
what does this fix, does that mean we don't support ios under 13 for any react native app?

Reply:
Nope, this was just to run the app locally.. I didn't have an older version of `iOS` to test against. This should allow any developer running macOS Ventura (13.0) with XCode to run the example app without any issue